### ISSUE TITLE
Abaout the bath size

### ISSUE THREAD

**Issue by Whishing** [qinzheng93/GeoTransformer#8]
Why the batch size is limited to 1 in the code?  And does the code support distributed training?

**Comment by qinzheng93** [comment]
Yes, the batch size is fixed to 1 per GPU for now. This is mainly because point clouds may have different numbers of points, so they are organized in *pack* mode. Supporting a batch size larger than 1 needs frequent conversion between *batch* and *pack*. I am considering supporting this but it could take some time.

An optional way to do this is to use distributed training. The code should already support distributed training, but I haven't tested it on GeoTransformer. See [base_trainer.py](https://github.com/qinzheng93/GeoTransformer/blob/73e1439a7a61cced595b67712f70e472bcbf76a4/geotransformer/engine/base_trainer.py#L32) for more details. You can try this on your own.

To use distributed training, just use `python -m torch.distributed.launch --nproc_per_node=4 trainval.py`. All settings in `config.py` is for one GPU, so you don't need to change it for distributed training. Note that **the learning rate will be multiplied by the world size**.

Hope this can help.
